tests/spectron/spellChecker.spectron.js

```diff
@@ -13,16 +12,14 @@ describe('Tests for spellChecker', () => {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
 
     beforeAll((done) => {
-        childProcess.exec(`npm rebuild robotjs --target=${process.version} --build-from-source`, () => {
-            robot = require('robotjs');
-            app.startApplication().then((startedApp) => {
-                app = startedApp;
-                done();
-            }).catch((err) => {
-                console.error(`Unable to start application error: ${err}`);
-                expect(err).toBeNull();
-                done();
-            });
+        robot = require('robotjs');
```

Comment:
You can require `robotjs` globally now 😄

Reply:
Done 😄